content/english/chatbot/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Create your own chatbot"
+description: "An introductory guide to chatbots"
+date: 2020-02-217T11:45:38-07:00
+prereq: "None"
+difficulty: "Beginner"
+download: ""
+draft: false
+icon: "fas fa-code"
+---
+## Introduction
+In this workshop, you will learn how to create your own chatbot. By the end of the workshop, you will have a chatbot that can have a simple conversation with the user. You will learn how to use AIML, a language for creating chatbots. You will also learn how to use Pandorabots, a platform for hosting chatbots.
+
+## What is a chatbot?
+A chatbot is a computer program that simulates human conversation to answer questions. 
+
+## Examples of chatbots
+Chances are you've already interacted with a chatbot as part of your daily life. 
+
+You might use a virtual assistant to set a reminder on your phone, or to play a song on your smart speaker.
+<p style="text-align: center; "><img src="./img/smart_device.jpg" alt="image of people talking to a smart speaker" width="500"/></p>
+
+You might have interacted with a chatbot to get help with a product or service.
+
+<p style="text-align: center; "><img src="./img/customer_service_chatbot.png" alt="new account landing page on github" width="500"/></p>
+
+More recently, chatbots have become more well-known thanks to technologies like ChatGPT.
+
+## History of chatbots
+Chatbots have been around for a long time. The first chatbot was created in 1966 and was called ELIZA. In 1995, a new chatbot called ALICE was created. ALICE was written using AIML, which is the same language we will learn in this workshop.
```

Comment:
Add a table of contents at the end of this page - see other workshops for examples